meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -15,4 +17,9 @@ do_install_append_lmp-wayland() {
 
     install -m 0644 ${WORKDIR}/utilities-terminal.png ${D}${datadir}/weston/icon/utilities-terminal.png
     install -m 0644 ${WORKDIR}/background.jpg ${D}${datadir}/weston/backgrounds/background.jpg
+    install -Dm 0644 ${WORKDIR}/weston.conf ${D}${sysconfdir}/default/weston
 }
+
+INI_UNCOMMENT_ASSIGNMENTS = " \
```

Comment:
This is a problem as this function is only available in meta-freescale, which can be removed by our customers (not every bsp layer is forced to be included, they are just by default).

```
meta-freescale/recipes-graphics/wayland/weston-init.bbappend

do_install_append() {
    for assignment in ${INI_UNCOMMENT_ASSIGNMENTS}; do
        uncomment "$assignment" ${D}${sysconfdir}/xdg/weston/weston.ini
    done
}
```

For having this generic to LmP we would need to have a similar function declared at meta-lmp-base, just do it in a way that still works correctly when meta-freescale is enabled and used.